config/job_config_sample.yaml

```diff
@@ -21,6 +21,8 @@ inputs:
       # Optional send any spark supported option to the reader
       options:
         quoteAll: false
+      # Optional define stream reader that can be used to read streaming data
```

Comment:
Optional, treat file path (must be a single path) as a streaming source, this will trigger jobs for new files added to the folder